catinfo/README.md

```diff
@@ -0,0 +1,5 @@
+# This README is all about cats!
+
+Did you know cats are cool? DUH. 
+
+
```

Comment:
Maybe add another fact here. Your filename implies there should be multiple facts in here.